load.go

```diff
@@ -196,6 +196,12 @@ func storeValue(rawValue string, valRv reflect.Value) error {
 	switch valRv.Kind() {
 	case reflect.String:
 		valRv.SetString(rawValue)
+	case reflect.Uint32:
+		value, err := strconv.ParseUint(rawValue, 10, 64)
```

Comment:
Shouldn't it be:

```suggestion
		value, err := strconv.ParseUint(rawValue, 10, 32)
```

?

Reply:
Or should we add a case for `Uint64`, and have `Uint32` `fallthrough` into it?

Reply:
yeah good call. I added the fallthrough and also unint, uint32 and int32 types too